docker-compose.yml.sample

```diff
@@ -0,0 +1,59 @@
+version: '2'
+
+services:
+  rails-api-base:
+    image: boilerplate/rails-api-base
+    build:
+      context: .
+    container_name: rails-api-base
+    volumes:
+      - ./app:/rails-api/app
+      - ./bin:/rails-api/bin
+      - ./config:/rails-api/config
+      - ./db:/rails-api/db
+      - ./lib:/rails-api/lib
+      - ./public:/rails-api/public
+      - ./config.ru:/rails-api/config.ru
+      - ./Rakefile:/rails-api/R
```

Comment:
I would rather let the user pass a command to docker run instead of hard-coding each possible environment with docker-compose's file.